sources/sql_database/__init__.py

```diff
@@ -142,6 +142,7 @@ def sql_table(
     table_adapter_callback: Callable[[Table], None] = None,
     backend_kwargs: Dict[str, Any] = None,
     type_adapter_callback: Optional[TTypeAdapter] = None,
+    included_columns: Optional[List[str]] = None,
```

Comment:
maybe we should just name it columns? WDYT?

Reply:
I wanted to but when there's a `COLUMNS` shell/env variable then everything fails. The config resolver tries to read that as a fallback when you don't set the option.